### DISCUSSION TITLE
Change the url of online store in Dashboard

### DISCUSSION THREAD

**Discussion by ashishkpaul** [saleor/saleor#10898] [Q&A] [answered]
**Wrong Invoice pdf url if you change Online store url in Dashbaord**
Question is releated to : [https://github.com/saleor/saleor/issues/6000](https://github.com/saleor/saleor/issues/6000)

I am using:
dashboard v3.7.8
core v3.7.22

But in dashboard **store general information is not visible**
[https://docs.saleor.io/docs/3.x/dashboard/configuration/site#general-information](https://docs.saleor.io/docs/3.x/dashboard/configuration/site#general-information)

**Comment by peelar** [comment]
Hello @ashishkpaul 👋 

In order for us to help you, you need to provide more details. Is this a bug? If so, please follow [the bug template](https://github.com/saleor/saleor/issues/new/choose). If not, please write a bit more context.

**Reply by ashishkpaul** [reply]
It seems to be bug. I wrote at https://github.com/saleor/saleor/issues/6000#issuecomment-1275964814.

**Reply by ashishkpaul** [reply]
Is there any update on this issue ????

**Comment by timuric** [comment, marked as answer]
As a workaround I would suggest to use site settings mutation to update url

**Reply by ashishkpaul** [reply]
Thanks sir, will try and update you

**Reply by ashishkpaul** [reply]
@timuric  Its done sir, Thanks
